### ISSUE TITLE
[Supportability] Consider feature to enable console tracing using Cookie

### ISSUE THREAD

**Issue by SergeyKanzhelev** [Microsoft/ApplicationInsights-JS#559]
In some cases it is hard to understand what is going on with Application Insights SDK. Console tracing may help troubleshoot the issue. But change of console trace level require re-instrumentation. 

I suggest to create an alternative way to change tracing level for Application Insights SDK using cookies or other runtime setting.

**Comment by KamilSzostak** [comment]
You can change the config object in dev tool. For instance: 

```
appInsights.config.verboseLogging = true;   
appInsights.config.enableDebug = true;
```

But you have to do it on every single page. Adding a flag to the cookie might be a good idea.

**Comment by SergeyKanzhelev** [comment]
Yes, loading issues is something particularly hard to troubleshoot.
